src/Magnum/DartIntegration/Object.h

```diff
@@ -40,72 +45,93 @@ namespace dart { namespace dynamics {
     class ShapeNode;
 }}
 
-namespace Magnum { namespace DartIntegration {
+namespace Magnum {
+
+class Mesh;
+class Buffer;
+template<UnsignedInt dimensions> class Texture;
```

Comment:
Please use one of the [forward declaration headers](http://doc.magnum.graphics/magnum/compilation-speedup.html#compilation-forward-declarations) instead of specifying the forward declarations by hand. It gets very messy very fast once the templates get involved :)

In this case it would be `#include <Magnum/Magnum.h>`.

Reply:
> Please use one of the forward declaration headers instead of specifying the forward declarations by hand. It gets very messy very fast once the templates get involved :)
> 
> In this case it would be #include <Magnum/Magnum.h>.

I should at some point read in more detail the docs! :smile: